helper.php

```diff
@@ -220,7 +220,7 @@ public function ensureAbsoluteId($id) {
      * @param Doku_Renderer_xhtml $R
```

Comment:
(Note: this are inline comments, maybe in the Github view two discussions are mixed)